doc/source/whatsnew/v1.0.0.rst

```diff
@@ -474,7 +474,7 @@ Other API changes
   Supplying anything else than ``how`` to ``**kwargs`` raised a ``TypeError`` previously (:issue:`29388`)
 - When testing pandas, the new minimum required version of pytest is 5.0.1 (:issue:`29664`)
 - :meth:`Series.str.__iter__` was deprecated and will be removed in future releases (:issue:`28277`).
-
+- Add ``ignore_index`` to :meth:`DataFrame.drop_duplicates` to reset index (:issue:`30114`)
```

Comment:
reverse the ordering here, e.g. .drop_duplicates has gained the `ignore_index` keyword.

move to other enhancements

Reply:
moved and rephrased!